core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/CredentialV4Controller.java

```diff
@@ -52,8 +50,7 @@
 
     @Override
     public CredentialV4Responses list(Long workspaceId) {
-        Set<Credential> credentials = credentialService.listAvailablesByWorkspaceId(workspaceId);
-        return new CredentialV4Responses(converterUtil.convertAllAsSet(credentials, CredentialV4Response.class));
+        return new CredentialV4Responses(converterUtil.convertAllAsSet(credentialService.listAvailablesByWorkspaceId(workspaceId), CredentialV4Response.class));
```

Comment:
i think it is not a problem if the credentialservice call is in a different line, it is more readable

Reply:
![images](https://user-images.githubusercontent.com/16577123/51121931-eb736200-1818-11e9-90fb-b2e054ad658d.jpeg)